ligo/main.mligo

```diff
@@ -13,6 +13,14 @@
 This is an example of conditionally present code, remove it once normal pragmas are set.
 #endif
 
+let get_registered_cumulatives_unsafe (buffer : timed_cumulatives_buffer) (i : nat) : timed_cumulatives =
+    match Big_map.find_opt i buffer.map with
+        | None -> (failwith internal_bad_access_to_observation_buffer)
+        | Some v -> v
+
+let get_last_cumulatives (buffer : timed_cumulatives_buffer) : timed_cumulatives =
+    get_registered_cumulatives_unsafe buffer buffer.last
```

Comment:
These are so simple that (from experience, I haven't checked) they may better be inlined.

In many cases paying for the execution cost is cheaper than paying for fetching, extracting and  executing a lambda from the bottom of the stack.

Reply:
Good catch :+1:  

This saves us more than a hundred bytes even now.

Reply:
:thinking: I see you've added it to `get_last_cumulatives`, but what about `get_registered_cumulatives_unsafe`?